src/plan_expand_hypertable.c

```diff
@@ -1172,6 +1178,7 @@ ts_plan_expand_hypertable_chunks(Hypertable *ht, PlannerInfo *root, RelOptInfo *
 		.all_quals = NIL,
 		.join_conditions = NIL,
 		.propagate_conditions = NIL,
+		.join_level = 0,
```

Comment:
This is the default for values not mentioned in the initializer, so strictly speaking not necessary.